content/en/docs/faq/acme.md

```diff
@@ -148,6 +148,8 @@ cert-manager will check if a DNS record has been propagated by querying the clus
 cert-manager by default uses SOA (Start of Authority) records to determine which zone name to use at your DNS provider.
 Some DNS resolvers will filter this information, if this is the case cert-manager cannot determine the zone and it is advised to [change the DNS server for DNS01 self-checks](../../configuration/acme/dns01/#setting-nameservers-for-dns01-self-check).
 
+If using dnsmasq as your local server this may occur if you use the flag `--filterwin2k`, the OpenWRT/LuCi enables this option if you check "filter useless queries". By enabling this flag, dnsmasq drops all SOA records.
```

Comment:
```suggestion
If you use `dnsmasq` as your DNS server, this may occur if you use the [`--filterwin2k` flag](www.thekelleys.org.uk/dnsmasq/docs/setup.html).
In [OpenWRT there is a `filterwin2k` configuration option](https://openwrt.org/docs/guide-user/base-system/dhcp#all_options). 
And in [LuCI there is a "Filter useless" option](https://github.com/openwrt/luci/blob/15757dd5b18f9e00ba3c9b38af4d46702a31fe33/modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js#L217-L219). 
By enabling this flag, `dnsmasq` drops all `SOA` records.
```